sql/core/pom.xml

```diff
@@ -223,6 +223,18 @@
       <artifactId>htmlunit3-driver</artifactId>
       <scope>test</scope>
     </dependency>
+    <!-- Explicit declaration of bouncy-castle dependencies are
+         needed for maven test builds on later hadoop releases.-->
```

Comment:
Oh, BTW, why do we need this in `sql/core` module, @steveloughran ?

Reply:
Currently, we have these test dependencies in YARN related module like the following.

https://github.com/apache/spark/blob/7e7ba4eaf0716676bc914706d8f6e2a61ee8b2c4/common/network-common/pom.xml#L164-L173

https://github.com/apache/spark/blob/7e7ba4eaf0716676bc914706d8f6e2a61ee8b2c4/resource-managers/yarn/pom.xml#L54-L65